cosmosdb/client_endpoint.bal

```diff
@@ -37,321 +38,232 @@ public isolated client class DataPlaneClient {
     #
     # + cosmosdbConfig - Configurations required to initialize the `Client` endpoint
     # + httpClientConfig - HTTP configuration
-    # + return -  Error at failure of client initialization
-    public isolated function init(ConnectionConfig cosmosdbConfig, http:ClientConfiguration httpClientConfig = {}) returns Error? {
+    # + return - Error at failure of client initialization
+    public isolated function init(ConnectionConfig cosmosdbConfig, http:ClientConfiguration httpClientConfig = {},
+    CustomizedConfiguration? customConfig = ()) returns Error? {
         self.baseUrl = cosmosdbConfig.baseUrl;
         self.primaryKeyOrResourceToken = cosmosdbConfig.primaryKeyOrResourceToken;
         self.host = getHost(cosmosdbConfig.baseUrl);
         self.httpClient = check new (self.baseUrl, httpClientConfig);
+        check initClient(self, cosmosdbConfig, customConfig);
     }
- 
+
     # Creates a document.
-    # 
+    #
     # + databaseId - ID of the database to which the container belongs to
     # + containerId - ID of the container where, document is created
+    # + documemtId - ID of the document
     # + document - A JSON document to be saved in the database
     # + partitionKey - The specific value related to the partition key field of the container 
-    # + documentCreateOptions - The `cosmos_db:DocumentCreateOptions` which can be used to add additional capabilities 
-    #                           to the request
-    # + return - If successful, returns `cosmos_db:Document`. Else returns `cosmos_db:Error`.
+    # + requestOptions - The `cosmos_db:DocumentCreateOptions` which can be used to add additional capabilities 
```

Comment:
Improve the description that we can override what we set at init

Reply:
Resolved in https://github.com/ballerina-platform/module-ballerinax-azure-cosmosdb/commit/e8efdeb8fbaf0bcf88e0b231605b1dc7df3b858e